src/components/AuthButton.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const styles = {
```

Comment:
why not using scss file.

Reply:
I didn't wanted to define a className, and for isolating the styles to only this component.